notebooks/official/custom/custom_training_tensorboard_profiler.ipynb

```diff
@@ -0,0 +1,1376 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    job = aiplatform.CustomPythonPackageTrainingJob(
</pre><p>If you are just packaging a single script, it's probably a lot easier to just use <code>CustomTrainingJob</code> instead of <code>CustomPythonPackageTrainingJob`.</code></p><p> </p><p>It might be nice to have an official example of CustomPythonPackageTrainingJob though, so feel free to keep it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/719/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='77'/>

Reply:
<p>Sure. Keep it as an example.</p>